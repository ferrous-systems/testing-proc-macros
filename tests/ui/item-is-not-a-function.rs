use contracts::contracts;

#[contracts]
struct S;

fn main() {}
