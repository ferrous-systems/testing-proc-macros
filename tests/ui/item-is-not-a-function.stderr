error: item is not a function

         = help: `#[contracts]` can only be used on functions

 --> $DIR/item-is-not-a-function.rs:4:1
  |
4 | struct S;
  | ^^^^^^^^^
